rpc_jobs/rpc_mtc.yml

```diff
@@ -0,0 +1,44 @@
+
+- project:
+    name: rpc-mtc-gate-trigger
+    repo_name: rpc-mtc
```

Comment:
Should this be changed to the repository's actual name?

Reply:
Yes, and all the other repo_name / repo fields.